server.js

```diff
@@ -99,6 +99,21 @@ app.post('/api/delete_word', function(req, res) {
     });
 });
 
+app.post('/api/delete_corpus', function(req, res) {
+    var username = req.query.username;
+    var password = req.query.password;
+
+    request.del('https://stream.watsonplatform.net/speech-to-text/api/v1/customizations/' + encodeURIComponent(req.query.customization_id) + '/corpora/' + encodeURIComponent(req.query.corpus))
```

Comment:
Lets change this to:
encodeURIComponent(req.query.corpus_name)
to match the api